src/test/unit/utils/RandomTest.cpp

```diff
@@ -414,7 +414,7 @@ void RandomTest::RunTests()
 
     {
       // choose 0 elements
-      UInt32 choices[0];
+      UInt32 choices[1];
```

Comment:
Why? Do you get a compiler error for an array of length 0? If so, we may just want to remove this particular test.